hack/test_e2e.sh

```diff
@@ -1134,6 +1143,9 @@ function help {
     printf "\t-B binary directory, specifies the path for the directory where binaries will be installed\n"
     printf "\t-D Dockerfile, specifies the path of the Dockerfile to use\n"
     printf "\t-E set E2E directory, specifies the path for the E2E directory\n"
+    printf "\t-m set the KPNG deployment model, can either be: \n\
+            * split-process-per-node [default] -> (To run KPNG server + client in separate containers/procsses per node)
+            * single-process-per-node [default] -> (To run KPNG server + Client in a single container/process per node)"
```

Comment:
How can both deployment models be `[default]`?
```
* split-process-per-node [default] ...
* single-process-per-node [default] ...
```

Reply:
Good point bad typo, thanks!